### ISSUE TITLE
a torrent error has ocurred (odroid-c2 aarch64 libreelec 8.2)

### ISSUE THREAD

**Issue by gentooza** [markop159/KODI-Popcorn-Time#95]
Hi,

I think this issue is related to [this one](https://github.com/markop159/KODI-Popcorn-Time/issues/86) but only with user saying is playing with an odroid-c2 board.

I think it's related with no having a torrent2http aarch64 executable, perhaps system is trying to use the arm one.

[I've found also](https://forum.libreelec.tv/thread/2271-mod-libreelec-for-s905-with-32-bit-libs-to-run-regular-arm-binaries/), libreelec is compiled in aarch64 with arm support, but it lacks the system libraries. 

[Also, I thinkk there are sollutions to bring up  torrent2http to aarch64 arch](https://raw.githubusercontent.com/afedchin/torrent2http/master/Makefile), but I'm not used to GO, etc. So I can't try myself.

Please, confirm.
and of course, thank you very much!

**Comment by markop159** [comment]
torrent2http used in this addon is precompiled and I did not compile it myself, I too am not familiar with GO. OSMC aarch64 build have 32 bit arm support. Maybe you can just try to add executable permissions to torrent2http executable file in arm folder or try to install system libraries needed to support arm 32 bit.

**Comment by gentooza** [comment]
Hi Marko!

thanks for the reply!
Now I'm not sure what's happend :-/
+ First I changed permissions of torrent2http executable (as you already said in the older post), but I couldn't execute it via ssh terminal neither.
+ I see no logs at all, I've set kodi to show me everything but I'm doing something wrong. So I'm totally blink guessing what is wrong
+ I've copied the torrent2http binary from [here](https://github.com/afedchin/t2h-binaries) to a resources/bin/linux_arm64, to a resources/bin/linux_aarch64 and also symlinked it to resources/bin/linux_arm but plugin keeps failing. Now at least, I can execute torrent2http executable via ssh terminal 

I don't know what tests do

cheers

**Comment by markop159** [comment]
For logs, on kodi go to Settings -> System -> Logging
under Debug Enable debug logging, now it should show on left upper corner log file location.

To see additional logs of addon, open addon settings and under Advanced -> Enable debugging.

If you happen to find a fix for your problem please share, so we can add this fix for others with same problem.

**Comment by gentooza** [comment]
well,

I've lost logs as I've reflashed libreelec to my odroid-c2. But it was an error with torrent2http.
what I've done is flashing a 32bit userspace version of libreelec, with this version I can execute arm binaries with the odroid-c2 (aarch64)
(perhaps was not compatible the version of torrent2http I took from binaries before with the version kodi popcorn time plugin was using :-S, it's older or newer)

now it's working perfectly.
I've found 32bits versions for the odroid-c2 in this [post](https://forum.odroid.com/viewtopic.php?f=144&t=27578) and this [repository](https://kszaq.libreelec.tv/s905/) 

thanks and good job with the plugin, it's really cool!
